Sources/JWTKit/JWTKeyCollection.swift

```diff
@@ -29,7 +29,7 @@ public actor JWTKeyCollection: Sendable {
         defaultJWTSerializer: some JWTSerializer = DefaultJWTSerializer(),
         logger: Logger = Logger(label: "jwt_kit_do_not_log", factory: { _ in SwiftLogNoOpLogHandler() })
     ) {
-        self.storage = [:]
+        storage = [:]
```

Comment:
Why change this? It's inconsistent with the rest of the lines

Reply:
Damn formatter